cmd/notation-azure-kv/main.go

```diff
@@ -1,27 +1,35 @@
 package main
 
 import (
+	"context"
 	"encoding/json"
 	"errors"
+	"fmt"
 	"os"
 
 	"github.com/Azure/notation-azure-kv/internal/version"
 	"github.com/notaryproject/notation-go/plugin/proto"
-	"github.com/urfave/cli/v2"
 )
 
 func main() {
-	app := &cli.App{
-		Name:    "notation-azure-kv",
-		Usage:   "Notation - Notary V2 Azure KV plugin",
-		Version: version.GetVersion(),
-		Commands: []*cli.Command{
-			metadataCommand,
-			signCommand,
-			describeKeyCommand,
-		},
+	if len(os.Args) < 2 {
+		help()
+		return
 	}
-	if err := app.Run(os.Args); err != nil {
+	ctx := context.Background()
+	var err error
+	switch os.Args[1] {
+	case string(proto.CommandGetMetadata):
+		runGetMetadata()
+	case string(proto.CommandDescribeKey):
+		err = runDescribeKey(ctx)
+	case string(proto.CommandGenerateSignature):
+		err = runSign(ctx)
```

Comment:
Instead of casting command to string type, we can do the other way around.

Reply:
Updated.